src/components/Comment/index.tsx

```diff
@@ -218,7 +262,7 @@ export default class CommentComponent extends React.Component<CommentProps> {
 
     const onSave = async (e: React.FormEvent) => {
       e.preventDefault();
-      await saveComment(comment, store);
+      await saveComment(comment, store, strings, true);
```

Comment:
In JS without named parameters, right-hand boolean values in function calls can be hard to understand, so this might be worth creating a `saveCommentForceWarning` method that just wraps `saveComment` so the code is easier to follow (nitpick really).